solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -253,262 +255,163 @@ describe("RandomBeacon - Relay", () => {
   })
 
   describe("submitRelayEntry", () => {
+    let requestStartBlock: number
```

Comment:
I think it is no longer used.

Reply:
See https://github.com/keep-network/keep-core/pull/2729/commits/ef1075cc3260578686539f43fba5bd11fc6b5a0a